src/definitionGenerator.js

```diff
@@ -39,6 +39,15 @@ class DefinitionGenerator {
 
     async parse() {
         this.createInfo()
+
+        if (this.serverless.service.custom.documentation.securitySchemes) {
+            this.createSecuritySchemes(this.serverless.service.custom.documentation.securitySchemes)
+
+            if (this.serverless.service.custom.documentation.overallSecurityRequirement) {
+                this.openAPI.security = this.serverless.service.custom.documentation.overallSecurityRequirement
```

Comment:
If you have line 277-280 of this file, then you dont need this conditional:

```
277-280
        if (Object.keys(documentation).includes('security')) {
            obj.security = documentation.security
        }
```

Reply:
Think you're misunderstanding here.  Line 46 is checking for a `security` (or previously `overallSecurityRequirement`) set within the documentation top level to apply to every operation.

Line 277 is checking on an operation level to see if there is a security config to override the overall security configuration set in Line 46.